bin/setup.sh

```diff
@@ -46,7 +47,7 @@ wait_for_plugin_registration () {
 }
 
 load_bundled_data ( ) {
-  __skriptz=$(ls -1 ${COOPR_HOME}/provisioner/lib/provisioner/worker/plugins/*/*/load-bundled-data.sh 2>&1)
+  __skriptz=$(ls -1 ${COOPR_PROVISIONER_PLUGIN_DIR}/*/*/load-bundled-data.sh 2>&1)
```

Comment:
good catch.  the original definition of ``COOPR_PROVISIONER_PLUGIN_DIR`` was incorrect (it was defined but never used).  will fix